_data/author.yml

```diff
@@ -5,4 +5,5 @@ aviaryan:
     twitter: aviaryan123
     tagline: Competitive programmer, Loves to build new things, Anime addict, Will travel the world some day
     link: https://aviaryan.in
-    img: https://github.com/aviaryan.png
\ No newline at end of file
+    img: https://github.com/aviaryan.png
+    
```

Comment:
This line contains whitespaces 😞

Reply:
![tenor](https://user-images.githubusercontent.com/56167998/94242536-cdd62580-ff33-11ea-89e1-e2cc62c8ab0b.gif)